src/api/posts/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@senecacdot/posts-service",
+  "version": "1.0.0",
+  "private": true,
+  "description": "A service for retrieving posts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": "Seneca-CDOT/telescope",
+  "license": "BSD-2-Clause",
+  "bugs": {
+    "url": "https://github.com/Seneca-CDOT/telescope/issues"
+  },
+  "homepage": "https://github.com/Seneca-CDOT/telescope#readme",
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "devDependencies": {
+    "del-cli": "^3.0.1",
+    "nodemon": "^2.0.7",
+    "pino-pretty": "^4.5.0"
+  },
+  "dependencies": {
+    "@elastic/elasticsearch": "^7.11.0",
+    "@elastic/elasticsearch-mock": "^0.3.0",
+    "@senecacdot/satellite": "^1.6.1",
+    "crypto": "^1.0.1",
```

Comment:
Crypto is built-into node.js now, you shouldn't need this.  Also, if the only reason you're pulling this in is to do the hash function, you can use what @Metropass added to Satellite:

```js
const { hash } = require('@senecacdot/satellite');
```